tools/cli-client/internals/cli/cli_main.go

```diff
@@ -55,7 +55,7 @@ func addHelp(p *flags.Parser) {
 func blockForConfirm(s string) error {
 	// check if is a tty
 	fi, err := os.Stdin.Stat()
-	if err != nil || fi.Mode()&os.ModeNamedPipe == 0 {
+	if err != nil && fi.Mode()&os.ModeNamedPipe == 1 {
```

Comment:
```suggestion
	if err != nil && fi.Mode()&os.ModeNamedPipe != 0 {
```
because os.ModeNamedPipe value is higher than 1, so "bit-wise and" with it will be higher than 1 as well.

Reply:
Good catch. Thanks!